### ISSUE TITLE
Label geographic coordinate to district coordinate 

### ISSUE THREAD

**Issue by juifa-tsai** [yennanliu/NYC_Taxi_Trip_Duration#4]
According to pre-EDA, the **pickup & dropoff locations** can be the significant variables for analysis. Those variables are stored with geographic coordinates, i.e. longitude v.s. latitude. However, those values may be non-meaningful and bias in **_regression method,_**, since they are for labeling the location mathematically instead for quantity. Thus, finding the meaningful & efficient way to label the location is an issue. Here is an basic idea - **Categorized to belonged districts instead of using continue value** : 

1. Category types/ranges : Boroughs (5) >  Community areas/boards, CB (18max/bor.) > Neighborhoods (?)
2. Label with encode method to binary bits, e.g  010 110 for one of CB (total 59). 

Discussion :
1. Should we use multi-categories & single category?
2. Use CB or Neighborhoods?
3. How to extract neighborhoods numbers?
3. How to map geographic coord. to particular district category?

**Comment by yennanliu** [comment]
Re :  
1-3. to -  discussion list 
4. you can google  "polygon ", there are already tools doing this, may be interesting if we can extract  features from lon & lat
maybe check :  
- http://fiware-orion.readthedocs.io/en/master/user/geolocation/
- https://rstudio-pubs-static.s3.amazonaws.com/257299_79c25a258f9c46e3ab2f3830653195dd.html
- https://stackoverflow.com/questions/31701101/how-do-i-get-region-district-polygon-boundaries-data-from-google-maps

for more information 
cheers!

**Comment by juifa-tsai** [comment]
Or we can define own regions without meaning?
For instance, let's cut the map to many square boxes?

**Comment by yennanliu** [comment]
yes ! we can, I did similar things before. we can separate the city into  group like " most busy",
"busy", "medium", and "casual"  ones.

**Comment by juifa-tsai** [comment]
Cool, I think it quite important do this before diving to ML.
I'd recommend do both: 
1. defining box region as chessboard -> for ML 
2. label as density -> make chessboard has meaning

What do you think?

I can try to build a function for defining the chessboard & linear distance in this weekend, if you don't mind the time delay lol
